src/Goerli-DssSpell.sol

```diff
@@ -16,29 +16,62 @@
 
 pragma solidity 0.6.12;
 // Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-// pragma experimental ABIEncoderV2;
+pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./Goerli-DssSpellCollateral.sol";
 
-interface RwaUrnLike {
-    function draw(uint256) external;
+import { VatAbstract } from "dss-interfaces/dss/VatAbstract.sol";
+import { JugAbstract } from "dss-interfaces/dss/JugAbstract.sol";
+
+interface CureAbstract {
+    function lift(address) external;
+}
+
+interface TeleportJoinLike {
+    function file(bytes32,bytes32,address) external;
+    function file(bytes32,bytes32,uint256) external;
+}
+
+interface TeleportRouterLike {
+    function file(bytes32,bytes32,address) external;
+}
+
+interface TeleportOracleAuthLike {
+    function file(bytes32,uint256) external;
+    function addSigners(address[] calldata) external;
+}
+
+interface EscrowLike {
+    function approve(address,address,uint256) external;
 }
 
 contract DssSpellAction is DssAction, DssSpellCollateralAction {
 
     // Provides a descriptive tag for bot consumption
     string public constant override description = "Goerli Spell";
 
-    uint256 constant internal MILLION  = 10 **  6;
-    uint256 constant internal BILLION  = 10 **  9;
-    uint256 constant internal WAD      = 10 ** 18;
+    address internal constant TELEPORT_JOIN = 0xAaEf3f1523291aDdaF98a8535c27F25971d823D2;
+    address internal constant ORACLE_AUTH = 0xD40ab915cB8232E8188e1a9137E4b5dCB86F0fd8;
+    address internal constant ROUTER = 0x75c2D0A33cB245acD1F10798fEdD4a42Be4951a9;
+    address internal constant LINEAR_FEE = 0x72Cb460888D401f991AB1a78ffc48EFcDcd155e8;
 
-    // address constant RWA008_A_OPERATOR       = 0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439;
-    // address constant RWA008_A_OUTPUT_CONDUIT = 0x1aA21d2E39EC0da185CA04609c8868bC324d8553;
-    uint256 constant RWA009_DRAW_AMOUNT      = 25_000_000 * WAD;
+    bytes32 constant internal ILK = "TELEPORT-FW-A";
```

Comment:
What does "FW" stand for?

Reply:
Fast Withdrawals - we will need to redeploy under a new ilk with the full version.

Reply:
Will we need to replace the `ilk` with the new version?

Reply:
this was discussed offline, resolving.